packages/codegen-ui-react/lib/__tests__/__snapshots__/studio-ui-codegen-react-forms.test.ts.snap

```diff
@@ -10626,7 +10630,13 @@ export default function MyPostForm(props) {
           });
           await DataStore.save(
             Post.copyOf(postRecord, (updated) => {
-              Object.assign(updated, modelFields);
+              Object.assign(updated, {
```

Comment:
the handling of both array and nonArray should also be in the create form of this model too - 2847.
(for the array one, looks like I left the update form out, and you fixed that)

Reply:
Ok. Ill fix that as well